tests/testsRunningInsideARM.txt

```diff
@@ -0,0 +1,37 @@
+Exceptions/ForeignThread/ForeignThreadExceptions/ForeignThreadExceptions.sh
```

Comment:
The runtest.sh usage says the following
```
--testDirFile=<path>             : Run tests only in the directories specified by the file at <path>.
                                   Paths are listed one line, relative to the directory specified by --testRootDir.
```

Also the check at line `1091` in the `tests/runtest.sh` file checks for the presence of the directory in the given file as follows:
```
if [ ! -d "$testDir" ]; then
   echo "Test directory does not exist: $testDir"
else
   run_tests_in_directory "./$testDir
fi
```

I believe the contents of this file should be replaced by the directories containing the scripts. For example,
`Exceptions/ForeignThread/ForeignThreadExceptions/` instead of `Exceptions/ForeignThread/ForeignThreadExceptions/ForeignThreadExceptions.sh`